test/Learners/LearnersTests/MatchboxRecommenderTests.cs

```diff
@@ -654,16 +657,35 @@ public void NativeDataFormatSerializationTest()
             // Train and serialize
             {
                 var recommender = this.CreateNativeDataFormatMatchboxRecommender();
-                recommender.Save(NotTrainedFileName);
+                using (var re = File.OpenWrite(NotTrainedFileName))
```

Comment:
I think I just preferred to call an overload that didn't take an IFormatter, and these take a Stream. But I guess if we're changing the BinaryReader to allow a null IFormatter (and throw if it is needed) then this can be reverted.

However, I will say that for debugging, the text writer/reader is much more convenient.

Reply:
So then change `Save` to do the more convenient thing.